circle.yml

```diff
@@ -27,29 +27,31 @@ dependencies:
       ls $HOME/miniconda/bin &&
       export PATH="$HOME/miniconda/bin:$PATH" &&
       source activate test &&
-      conda install -c spyder-ide pytest-cov &&
+      conda install -q -c spyder-ide pytest-cov &&
+      conda install -q spyder &&
+      conda remove -y spyder &&
```

Comment:
@jitseniesen just FYI, the ability to install `--only-deps` will be [coming](https://github.com/conda/conda/pull/5265) with conda 4.4, apparently.